### PR TITLE
docs: update Halo deployment documentation

### PR DESCRIPTION
**Pull request by ruibaby**
Updated Halo documentation to use templates for deployment

### CODE REVIEW

File: docs/pages/en-US/marketplace/halo.mdx [hunk 1]
```diff
@@ -1,78 +1,81 @@
 ---
 title: Deploy Halo
 ogImageTitle: Deploy Halo
-ogImageSubtitle: Deploy Halo on Zeabur
+ogImageSubtitle: One-click Deploy Halo Service on Zeabur
```
Comment (pan93412) [inline docs/pages/en-US/marketplace/halo.mdx:4]:
```suggestion
ogImageSubtitle: Deploy Halo Service on Zeabur with One Click
```

File: docs/pages/en-US/marketplace/halo.mdx [hunk 2]
```diff
@@ -1,78 +1,81 @@
...
+# Deploy Halo
 
-[Halo](https://halo.run/) is a powerful and easy-to-use open source website building tool, with rich themes and plugins to help you build the ideal site in your mind.
+[Halo](https://www.halo.run/) is a powerful and easy-to-use open-source website building tool. With a rich set of templates and plugins, it helps you build your ideal website.
 
-This article will teach you how to deploy your own Halo service on Zeabur through the service market, and bind your own domain name.
+This article will guide you on how to easily deploy your own Halo service on Zeabur through the service marketplace and bind your own domain name.
 
-## Step 1: Create a project
+## Deploy Halo from Template
```
Comment (pan93412) [inline docs/pages/en-US/marketplace/halo.mdx:15]: This subtitle may be unnecessary. Perhaps you can incorporate the steps into Title 2 (`##`) instead.

Comment (ruibaby) [reply docs/pages/en-US/marketplace/halo.mdx]: Updated

**Review by pan93412 — APPROVED** [review]
I will improve the traditional chinese translation later.

LGTM, thanks!

**Review by yuaanlin — APPROVED** [review]

**Pull request merged**